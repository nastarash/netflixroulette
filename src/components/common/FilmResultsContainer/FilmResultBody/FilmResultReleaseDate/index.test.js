import React from 'react';
import { FilmResultReleaseDate } from './index';

describe('<FilmResultReleaseDate />', () => {
  test('should be rendered correctly', () => {
    const filmResultReleaseDate = shallow(
      <FilmResultReleaseDate date={2000} />
    );
    expect(filmResultReleaseDate).toMatchSnapshot();
  });

  describe('the rendered date should be consistent', () => {
    test('year 2000 rendered as 2000', () => {
      const filmResultReleaseDate = shallow(
        <FilmResultReleaseDate date={2000} />
      );
      expect(filmResultReleaseDate.text()).toEqual('2000');
    });

    test('year 1990 rendered as 1990', () => {
      const filmResultReleaseDate = shallow(
        <FilmResultReleaseDate date={1990} />
      );
      expect(filmResultReleaseDate.text()).toEqual('1990');
    });
  });
});