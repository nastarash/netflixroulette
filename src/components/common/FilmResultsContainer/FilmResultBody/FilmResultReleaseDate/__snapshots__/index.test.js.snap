// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FilmResultReleaseDate /> should be rendered correctly 1`] = `
<Fragment>
  2000
</Fragment>
`;
